docs/_themes/clusterhq/layout_docs.html

```diff
@@ -256,13 +251,25 @@
 
             // Is version equal to this documentation release?
             if (currentversion !== DOCUMENTATION_OPTIONS.VERSION) {
+                current_url = window.location.pathname
+                new_url = current_url.replace(
+                    DOCUMENTATION_OPTIONS.VERSION, currentversion)
                 $('.section:first').prepend('<div class="admonition note warning"> \
                     <p class="last"> \
                         You are currently viewing a different version of the documentation to the latest release of Flocker. \
-                        The latest version is ' + currentversion + ', you are viewing ' + DOCUMENTATION_OPTIONS.VERSION + '.</p> \
+                        The latest version is ' + currentversion + ', you are viewing ' + DOCUMENTATION_OPTIONS.VERSION + '. <a href="' + new_url + '">Click here to view the latest version</a>.</p> \
```

Comment:
This change aims to add links to the latest version. It needs testing, but I am not sure how to do this.